### DISCUSSION TITLE
airflow command error: invalid choice: upgrade_check

### DISCUSSION THREAD

**Discussion by ginnyedelstein** [apache/airflow#13369] [Q&A] [answered]
I've been unable to run the `airflow upgrade_check` command - getting invalid choice with:
pip==20.2.4
apache-airflow==2.0.0
apache-airflow-upgrade-check==1.1.0

**Comment by potiuk** [comment, marked as answer]
This is correct. the 'airflow-upgrade-check' should only be run next to airflow 1.10.* (preferably bridge 1.10.14 release) not Airflow 2.0. It should be run BEFORE you upgrade to 2.0.

**Reply by ginnyedelstein** [reply]
thank you!!
